### PR TITLE
feat: add share/unshare volume to the message bus

### PR DESCRIPTION
**Pull request by paulyoong**
Allow share/unshare volume requests to be made via the message bus.

### CODE REVIEW

**Comment by paulyoong** [comment]
bors try

File: common/src/mbus_api/message_bus/v0.rs [hunk 1]
```diff
@@ -254,6 +255,20 @@ pub trait MessageBusTrait: Sized {
         let request = SetVolumeReplica::new(uuid, replica);
         Ok(request.request().await?)
     }
+    /// share volume
```
Comment (tiagolobocastro) [inline common/src/mbus_api/message_bus/v0.rs:258]: add new line above?

**Review by tiagolobocastro — APPROVED** [review]

File: common/src/mbus_api/message_bus/v0.rs (hunk 1 above)
Comment (paulyoong) [reply common/src/mbus_api/message_bus/v0.rs:259]: Done.

**Comment by paulyoong** [comment]
bors merge

**Pull request merged**